include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -3827,7 +3836,8 @@ MGTwoLevelTransferNonNested<dim, LinearAlgebra::distributed::Vector<Number>>::
   Assert(dof_handler_coarse.get_fe().n_components() > 0 &&
            dof_handler_fine.get_fe().n_components() > 0,
          ExcNotImplemented());
-  Assert(dof_handler_fine.n_dofs() > dof_handler_coarse.n_dofs(),
+  Assert(additional_data.allow_more_dofs_in_coarse_dof_handler ||
+           dof_handler_fine.n_dofs() > dof_handler_coarse.n_dofs(),
```

Comment:
```suggestion
```

Reply:
Sure, if this is OK :)
I thought I would be the only one who might need this, and that it is not justified to delete the Assert because of some corner cases. I will adjust the code :)